service-rpc/src/main/java/org/apache/iotdb/rpc/Config.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.iotdb.rpc;
+
+
+/**
+ * @author chenPeng
+ * @version 1.0.0
+ * @ClassName Config.java
+ * @Description TODO
+ * @createTime 2020年08月19日 22:26:00
+ */
+public class Config {
```

Comment:
Please reformat this class and avoid use Chinese in Javadoc.

Reply:
```suggestion
public class Config {
```